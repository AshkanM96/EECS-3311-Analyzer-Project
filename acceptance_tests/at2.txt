divide 
reset
divide
integer_constant (5)
integer_constant (0)
type_check
evaluate
reset
