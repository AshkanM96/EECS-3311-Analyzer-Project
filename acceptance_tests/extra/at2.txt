intersect
open_set
int(3)
divide
int(2)
count
open_set
bool(false)
lt
close_set
mod
int(3)
int(0)
int(909)
close_set
int(3)
int(4)
close_set
open_set
int(1)
int(3)
close_set
evaluate
type_check
reset
union
open_set
int(3)
divide
int(2)
count
open_set
bool(false)
lt
close_set
mod
int(3)
int(0)
int(909)
close_set
int(3)
int(4)
close_set
open_set
int(1)
int(3)
close_set
evaluate
type_check
reset
difference
open_set
int(3)
divide
int(2)
count
open_set
bool(false)
lt
close_set
mod
int(3)
int(0)
int(909)
close_set
int(3)
int(4)
close_set
open_set
int(1)
int(3)
close_set
evaluate
type_check
reset
difference
open_set
int(1)
int(3)
close_set
open_set
int(3)
divide
int(2)
count
open_set
bool(false)
lt
close_set
mod
int(3)
int(0)
int(909)
close_set
int(3)
int(4)
close_set
evaluate
type_check
reset
subset_equal
open_set
int(3)
divide
int(2)
count
open_set
bool(false)
lt
close_set
mod
int(3)
int(0)
int(909)
close_set
int(3)
int(4)
close_set
open_set
int(1)
int(3)
close_set
evaluate
type_check
reset
subset_proper
open_set
int(3)
divide
int(2)
count
open_set
bool(false)
lt
close_set
mod
int(3)
int(0)
int(909)
close_set
int(3)
int(4)
close_set
open_set
int(1)
int(3)
close_set
evaluate
type_check
reset
superset_equal
open_set
int(3)
divide
int(2)
count
open_set
bool(false)
lt
close_set
mod
int(3)
int(0)
int(909)
close_set
int(3)
int(4)
close_set
open_set
int(1)
int(3)
close_set
evaluate
type_check
reset
superset_proper
open_set
int(1)
int(3)
close_set
open_set
int(3)
divide
int(2)
count
open_set
bool(false)
lt
close_set
mod
int(3)
int(0)
int(909)
close_set
int(3)
int(4)
close_set
evaluate
type_check
reset
