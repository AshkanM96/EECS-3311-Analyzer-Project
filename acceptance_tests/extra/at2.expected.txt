  Expression currently specified: ?
  Report: Expression is initialized.
->intersect
  Expression currently specified: (? /\ nil)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({?} /\ nil)
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({3, ?} /\ nil)
  Report: OK.
->divide
  Expression currently specified: ({3, (? / nil)} /\ nil)
  Report: OK.
->integer_constant(2)
  Expression currently specified: ({3, (2 / ?)} /\ nil)
  Report: OK.
->count
  Expression currently specified: ({3, (2 / (# ?))} /\ nil)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({3, (2 / (# {?}))} /\ nil)
  Report: OK.
->boolean_constant(False)
  Expression currently specified: ({3, (2 / (# {False, ?}))} /\ nil)
  Report: OK.
->less_than
  Expression currently specified: ({3, (2 / (# {False, (? < nil)}))} /\ nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, (? < nil)}))} /\ nil)
  Report: Error (Set enumeration is not being specified).
->mod
  Expression currently specified: ({3, (2 / (# {False, ((? mod nil) < nil)}))} /\ nil)
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod ?) < nil)}))} /\ nil)
  Report: OK.
->integer_constant(0)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < ?)}))} /\ nil)
  Report: OK.
->integer_constant(909)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909), ?}))} /\ nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), ?} /\ nil)
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, ?} /\ nil)
  Report: OK.
->integer_constant(4)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4, ?} /\ nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} /\ ?)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} /\ {?})
  Report: OK.
->integer_constant(1)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} /\ {1, ?})
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} /\ {1, 3, ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} /\ {1, 3})
  Report: OK.
->evaluate
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} /\ {1, 3})
  Report: Error (Modulus is not positive).
->type_check
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} /\ {1, 3})
  Report: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} /\ {1, 3}) is type-correct.
->reset
  Expression currently specified: ?
  Report: OK.
->union
  Expression currently specified: (? \/ nil)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({?} \/ nil)
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({3, ?} \/ nil)
  Report: OK.
->divide
  Expression currently specified: ({3, (? / nil)} \/ nil)
  Report: OK.
->integer_constant(2)
  Expression currently specified: ({3, (2 / ?)} \/ nil)
  Report: OK.
->count
  Expression currently specified: ({3, (2 / (# ?))} \/ nil)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({3, (2 / (# {?}))} \/ nil)
  Report: OK.
->boolean_constant(False)
  Expression currently specified: ({3, (2 / (# {False, ?}))} \/ nil)
  Report: OK.
->less_than
  Expression currently specified: ({3, (2 / (# {False, (? < nil)}))} \/ nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, (? < nil)}))} \/ nil)
  Report: Error (Set enumeration is not being specified).
->mod
  Expression currently specified: ({3, (2 / (# {False, ((? mod nil) < nil)}))} \/ nil)
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod ?) < nil)}))} \/ nil)
  Report: OK.
->integer_constant(0)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < ?)}))} \/ nil)
  Report: OK.
->integer_constant(909)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909), ?}))} \/ nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), ?} \/ nil)
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, ?} \/ nil)
  Report: OK.
->integer_constant(4)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4, ?} \/ nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} \/ ?)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} \/ {?})
  Report: OK.
->integer_constant(1)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} \/ {1, ?})
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} \/ {1, 3, ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} \/ {1, 3})
  Report: OK.
->evaluate
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} \/ {1, 3})
  Report: Error (Modulus is not positive).
->type_check
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} \/ {1, 3})
  Report: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} \/ {1, 3}) is type-correct.
->reset
  Expression currently specified: ?
  Report: OK.
->difference
  Expression currently specified: (? \ nil)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({?} \ nil)
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({3, ?} \ nil)
  Report: OK.
->divide
  Expression currently specified: ({3, (? / nil)} \ nil)
  Report: OK.
->integer_constant(2)
  Expression currently specified: ({3, (2 / ?)} \ nil)
  Report: OK.
->count
  Expression currently specified: ({3, (2 / (# ?))} \ nil)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({3, (2 / (# {?}))} \ nil)
  Report: OK.
->boolean_constant(False)
  Expression currently specified: ({3, (2 / (# {False, ?}))} \ nil)
  Report: OK.
->less_than
  Expression currently specified: ({3, (2 / (# {False, (? < nil)}))} \ nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, (? < nil)}))} \ nil)
  Report: Error (Set enumeration is not being specified).
->mod
  Expression currently specified: ({3, (2 / (# {False, ((? mod nil) < nil)}))} \ nil)
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod ?) < nil)}))} \ nil)
  Report: OK.
->integer_constant(0)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < ?)}))} \ nil)
  Report: OK.
->integer_constant(909)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909), ?}))} \ nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), ?} \ nil)
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, ?} \ nil)
  Report: OK.
->integer_constant(4)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4, ?} \ nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} \ ?)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} \ {?})
  Report: OK.
->integer_constant(1)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} \ {1, ?})
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} \ {1, 3, ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} \ {1, 3})
  Report: OK.
->evaluate
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} \ {1, 3})
  Report: Error (Modulus is not positive).
->type_check
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} \ {1, 3})
  Report: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} \ {1, 3}) is type-correct.
->reset
  Expression currently specified: ?
  Report: OK.
->difference
  Expression currently specified: (? \ nil)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({?} \ nil)
  Report: OK.
->integer_constant(1)
  Expression currently specified: ({1, ?} \ nil)
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({1, 3, ?} \ nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({1, 3} \ ?)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({1, 3} \ {?})
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({1, 3} \ {3, ?})
  Report: OK.
->divide
  Expression currently specified: ({1, 3} \ {3, (? / nil)})
  Report: OK.
->integer_constant(2)
  Expression currently specified: ({1, 3} \ {3, (2 / ?)})
  Report: OK.
->count
  Expression currently specified: ({1, 3} \ {3, (2 / (# ?))})
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({1, 3} \ {3, (2 / (# {?}))})
  Report: OK.
->boolean_constant(False)
  Expression currently specified: ({1, 3} \ {3, (2 / (# {False, ?}))})
  Report: OK.
->less_than
  Expression currently specified: ({1, 3} \ {3, (2 / (# {False, (? < nil)}))})
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({1, 3} \ {3, (2 / (# {False, (? < nil)}))})
  Report: Error (Set enumeration is not being specified).
->mod
  Expression currently specified: ({1, 3} \ {3, (2 / (# {False, ((? mod nil) < nil)}))})
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({1, 3} \ {3, (2 / (# {False, ((3 mod ?) < nil)}))})
  Report: OK.
->integer_constant(0)
  Expression currently specified: ({1, 3} \ {3, (2 / (# {False, ((3 mod 0) < ?)}))})
  Report: OK.
->integer_constant(909)
  Expression currently specified: ({1, 3} \ {3, (2 / (# {False, ((3 mod 0) < 909), ?}))})
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({1, 3} \ {3, (2 / (# {False, ((3 mod 0) < 909)})), ?})
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({1, 3} \ {3, (2 / (# {False, ((3 mod 0) < 909)})), 3, ?})
  Report: OK.
->integer_constant(4)
  Expression currently specified: ({1, 3} \ {3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4, ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({1, 3} \ {3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4})
  Report: OK.
->evaluate
  Expression currently specified: ({1, 3} \ {3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4})
  Report: Error (Modulus is not positive).
->type_check
  Expression currently specified: ({1, 3} \ {3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4})
  Report: ({1, 3} \ {3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4}) is type-correct.
->reset
  Expression currently specified: ?
  Report: OK.
->subset_equal
  Expression currently specified: (? <= nil)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({?} <= nil)
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({3, ?} <= nil)
  Report: OK.
->divide
  Expression currently specified: ({3, (? / nil)} <= nil)
  Report: OK.
->integer_constant(2)
  Expression currently specified: ({3, (2 / ?)} <= nil)
  Report: OK.
->count
  Expression currently specified: ({3, (2 / (# ?))} <= nil)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({3, (2 / (# {?}))} <= nil)
  Report: OK.
->boolean_constant(False)
  Expression currently specified: ({3, (2 / (# {False, ?}))} <= nil)
  Report: OK.
->less_than
  Expression currently specified: ({3, (2 / (# {False, (? < nil)}))} <= nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, (? < nil)}))} <= nil)
  Report: Error (Set enumeration is not being specified).
->mod
  Expression currently specified: ({3, (2 / (# {False, ((? mod nil) < nil)}))} <= nil)
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod ?) < nil)}))} <= nil)
  Report: OK.
->integer_constant(0)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < ?)}))} <= nil)
  Report: OK.
->integer_constant(909)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909), ?}))} <= nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), ?} <= nil)
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, ?} <= nil)
  Report: OK.
->integer_constant(4)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4, ?} <= nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} <= ?)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} <= {?})
  Report: OK.
->integer_constant(1)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} <= {1, ?})
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} <= {1, 3, ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} <= {1, 3})
  Report: OK.
->evaluate
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} <= {1, 3})
  Report: Error (Modulus is not positive).
->type_check
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} <= {1, 3})
  Report: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} <= {1, 3}) is type-correct.
->reset
  Expression currently specified: ?
  Report: OK.
->subset_proper
  Expression currently specified: (? < nil)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({?} < nil)
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({3, ?} < nil)
  Report: OK.
->divide
  Expression currently specified: ({3, (? / nil)} < nil)
  Report: OK.
->integer_constant(2)
  Expression currently specified: ({3, (2 / ?)} < nil)
  Report: OK.
->count
  Expression currently specified: ({3, (2 / (# ?))} < nil)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({3, (2 / (# {?}))} < nil)
  Report: OK.
->boolean_constant(False)
  Expression currently specified: ({3, (2 / (# {False, ?}))} < nil)
  Report: OK.
->less_than
  Expression currently specified: ({3, (2 / (# {False, (? < nil)}))} < nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, (? < nil)}))} < nil)
  Report: Error (Set enumeration is not being specified).
->mod
  Expression currently specified: ({3, (2 / (# {False, ((? mod nil) < nil)}))} < nil)
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod ?) < nil)}))} < nil)
  Report: OK.
->integer_constant(0)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < ?)}))} < nil)
  Report: OK.
->integer_constant(909)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909), ?}))} < nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), ?} < nil)
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, ?} < nil)
  Report: OK.
->integer_constant(4)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4, ?} < nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} < ?)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} < {?})
  Report: OK.
->integer_constant(1)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} < {1, ?})
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} < {1, 3, ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} < {1, 3})
  Report: OK.
->evaluate
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} < {1, 3})
  Report: Error (Modulus is not positive).
->type_check
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} < {1, 3})
  Report: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} < {1, 3}) is type-correct.
->reset
  Expression currently specified: ?
  Report: OK.
->superset_equal
  Expression currently specified: (? >= nil)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({?} >= nil)
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({3, ?} >= nil)
  Report: OK.
->divide
  Expression currently specified: ({3, (? / nil)} >= nil)
  Report: OK.
->integer_constant(2)
  Expression currently specified: ({3, (2 / ?)} >= nil)
  Report: OK.
->count
  Expression currently specified: ({3, (2 / (# ?))} >= nil)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({3, (2 / (# {?}))} >= nil)
  Report: OK.
->boolean_constant(False)
  Expression currently specified: ({3, (2 / (# {False, ?}))} >= nil)
  Report: OK.
->less_than
  Expression currently specified: ({3, (2 / (# {False, (? < nil)}))} >= nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, (? < nil)}))} >= nil)
  Report: Error (Set enumeration is not being specified).
->mod
  Expression currently specified: ({3, (2 / (# {False, ((? mod nil) < nil)}))} >= nil)
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod ?) < nil)}))} >= nil)
  Report: OK.
->integer_constant(0)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < ?)}))} >= nil)
  Report: OK.
->integer_constant(909)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909), ?}))} >= nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), ?} >= nil)
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, ?} >= nil)
  Report: OK.
->integer_constant(4)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4, ?} >= nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} >= ?)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} >= {?})
  Report: OK.
->integer_constant(1)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} >= {1, ?})
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} >= {1, 3, ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} >= {1, 3})
  Report: OK.
->evaluate
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} >= {1, 3})
  Report: Error (Modulus is not positive).
->type_check
  Expression currently specified: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} >= {1, 3})
  Report: ({3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4} >= {1, 3}) is type-correct.
->reset
  Expression currently specified: ?
  Report: OK.
->superset_proper
  Expression currently specified: (? > nil)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({?} > nil)
  Report: OK.
->integer_constant(1)
  Expression currently specified: ({1, ?} > nil)
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({1, 3, ?} > nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({1, 3} > ?)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({1, 3} > {?})
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({1, 3} > {3, ?})
  Report: OK.
->divide
  Expression currently specified: ({1, 3} > {3, (? / nil)})
  Report: OK.
->integer_constant(2)
  Expression currently specified: ({1, 3} > {3, (2 / ?)})
  Report: OK.
->count
  Expression currently specified: ({1, 3} > {3, (2 / (# ?))})
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({1, 3} > {3, (2 / (# {?}))})
  Report: OK.
->boolean_constant(False)
  Expression currently specified: ({1, 3} > {3, (2 / (# {False, ?}))})
  Report: OK.
->less_than
  Expression currently specified: ({1, 3} > {3, (2 / (# {False, (? < nil)}))})
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({1, 3} > {3, (2 / (# {False, (? < nil)}))})
  Report: Error (Set enumeration is not being specified).
->mod
  Expression currently specified: ({1, 3} > {3, (2 / (# {False, ((? mod nil) < nil)}))})
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({1, 3} > {3, (2 / (# {False, ((3 mod ?) < nil)}))})
  Report: OK.
->integer_constant(0)
  Expression currently specified: ({1, 3} > {3, (2 / (# {False, ((3 mod 0) < ?)}))})
  Report: OK.
->integer_constant(909)
  Expression currently specified: ({1, 3} > {3, (2 / (# {False, ((3 mod 0) < 909), ?}))})
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({1, 3} > {3, (2 / (# {False, ((3 mod 0) < 909)})), ?})
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({1, 3} > {3, (2 / (# {False, ((3 mod 0) < 909)})), 3, ?})
  Report: OK.
->integer_constant(4)
  Expression currently specified: ({1, 3} > {3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4, ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({1, 3} > {3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4})
  Report: OK.
->evaluate
  Expression currently specified: ({1, 3} > {3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4})
  Report: Error (Modulus is not positive).
->type_check
  Expression currently specified: ({1, 3} > {3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4})
  Report: ({1, 3} > {3, (2 / (# {False, ((3 mod 0) < 909)})), 3, 4}) is type-correct.
->reset
  Expression currently specified: ?
  Report: OK.
